### PR TITLE
Astro 5/30 Release notes

### PR DESCRIPTION
**Pull request by jwitz**
Might cut custom operator note if it's not ready to launch, but wanted to have it up just in case

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+### Manage permissions for groups of users with Teams
+
+Configure Teams from the Cloud UI to manage the permissions for many users across Workspaces from a single page. _Teams_ are a group of users in an Organization that you grant the same Workspace permissions, without needing to define them individually. 
+
+See [Make a Team](add-users.md#make-a-team) for setup steps.
+
+### Use Airflow operators in the Astro Cloud IDE
+
+You can now configure _operator cells_ in the Astro Cloud IDE to add Airflow operator functionality to your pipelines. You can create an operator cell based on any available Airflow operator in the [Astronomer Registry](https://registry.astronomer.io/), or you can create a custom operator cell type to run your team's custom operators. See [LINK] for more information.
```
Comment (lzdanski) [inline astro/release-notes.md:34]: You've got a [LINK] that's blocked by https://github.com/astronomer/docs/pull/2334/files

**Review by lzdanski — COMMENTED** [review]
Just one thing!

**Review by cmarteepants — APPROVED** [review]

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,23 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+### Manage permissions for groups of users with Teams
+
+Configure Teams from the Cloud UI to manage the permissions for many users across Workspaces from a single page. _Teams_ are a group of users in an Organization that you grant the same Workspace permissions, without needing to define them individually. 
+
+See [Make a Team](add-users.md#make-a-team) for setup steps.
+
+### Use Airflow operators in the Astro Cloud IDE
+
+You can now configure _operator cells_ in the Astro Cloud IDE to add Airflow operator functionality to your pipelines. You can create an operator cell based on any available Airflow operator in the [Astronomer Registry](https://registry.astronomer.io/), or you can create a custom operator cell type to run your team's custom operators. See [LINK] for more information. 
+
```
Comment (jwitz) [inline astro/release-notes.md:35]:
```suggestion
```

File: astro/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,19 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+### Manage permissions for groups of users with Teams
+
+Configure Teams from the Cloud UI to manage the permissions for many users across Workspaces from a single page. _Teams_ are a group of users in an Organization that you grant the same Workspace permissions, without needing to define them individually. 
+
+See [Make a Team](add-users.md#make-a-team) for setup steps.
+
+### Bug fixes
+
+- In Astro Hosted, removed an irrelevant **AWS external ID** info page from the Cloud UI.
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
- In Astro Hosted, an irrelevant **AWS external ID** info page has been removed from the Cloud UI.
```

**Comment by chomipi88** [comment]
Forgot to mention we are going to go `Public Preview` with Audit Logs once the below PR is merged as well. We can include that here or wait till next week.

https://github.com/astronomer/docs/pull/2339

**Pull request merged**